test/Microsoft.Azure.Devices.Edge.Test.Common/TomlDocument.cs

```diff
@@ -86,5 +86,23 @@ void AddTable<T>(string tableName, string key, T value)
                 $"[{tableName}]\n" +
                 $"{key} = {v}\n");
         }
+
+        public void AddToTableWithExistingEntry<T>(string tableName, string key, T value)
+        {
+            // Split the TOML table contents on tablename to check for an already existing entry.
+            string bracketTableName = "[" + tableName + "]";
+            string[] segments = this.document.ToString().Split(bracketTableName);
+            if (segments.Length > 1)
+            {
+                string v = value is string ? $"\"{value}\"" : value.ToString().ToLower();
+                string addedString = $"[{tableName}]\n" + $"{key} = {v}";
```

Comment:
Could be `${bracketTableName} \n" + $"{key} = {v}"`?

Reply:
It doesn't make a big difference :)